PSDepend/Private/Find-NugetPackage.ps1

```diff
@@ -12,27 +12,39 @@ function Find-NugetPackage {
         #If specified takes precedence over version
         [switch]$IsLatest,
 
-        [string]$Version
+		[string]$Version,
+
+		# If specified, gets passed during the Nuget source call
+		[pscredential]$Credential = $null
     )
 
     #Ugly way to do this.  Prefer islatest, otherwise look for version, otherwise grab all matching modules
-    if($IsLatest)
-    {
+	if($IsLatest)
```

Comment:
I think this is in a few spots - can you replace tabs with 4 spaces?  Thanks!

Reply:
I replaced all the tabs in this file